ods_ci/tests/Tests/0100__platform/0101__deploy/0104__operators/0104__rhods_operator/0114__component_images.robot

```diff
@@ -0,0 +1,104 @@
+*** Settings ***
+Documentation       Test Suite to check for components
+
+Resource            ../../../../../../tasks/Resources/RHODS_OLM/install/oc_install.robot
+
+Suite Setup         Suite Setup
+Suite Teardown      Suite Teardown
+
+
+*** Variables ***
+@{COMPONENTS} =
+...                 codeflare
+...                 modelregistry
+...                 trustyai
+...                 ray
+...                 kueue
+...                 workbenches
+...                 dashboard
+...                 modelmeshserving
+...                 datasciencepipelines
+...                 trainingoperator
+
+
+*** Test Cases ***
+Check For Correct Component Images
+    [Documentation]    The purpose is to enforce the check of correct component images on their deployments.
+    [Tags]                  Operator                RHOAIENG-12576
```

Comment:
Can we add a suite tag? maybe Smoke or Tier1

Reply:
sure, I think Smoke will fit as it is a quick check